operations/node_management/Change_Settings_in_the_Bond.md

```diff
@@ -0,0 +1,41 @@
+## Change Settings in the Bond
+
+iPXE is used to setup udev rules for interface names and bond members. The configuration of these are dynamic on boot until node customization runs (cloud-init) and sets up the conventional `/etc/sysconfig/network/ifcfg-bond0` and other neighboring files.
+
+The initial settings of the bond(s) can be changed directly in the LiveCD or with the Boot Script Service (BSS). When cabling is different than normal, there is flexibility for customizing bond links.
+
+
+### Prerequisites
+
+This procedure requires administrative privileges.
+
+
+### Procedure
+
+1. Customize the settings of the bond(s).
+   
+   For more information, refer to the following lines of the `script.ipxe` file in the metal-ipxe repository:
```

Comment:
final nit: hyper-link https://github.com/Cray-HPE/metal-ipxe/blob/main/script.ipxe to "metal-ipxe repository"